spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -1,8 +1,185 @@
 [[overview]]
-== Spring Cloud Stream Modules 
 
-This section goes into more detail about how you can work with Spring Cloud Stream Module as standalone applications or with Spring Cloud Data Flow.
+This section will provide you with a detailed overview of Spring Cloud Stream Application Starters, their purpose, and how to use them.
+It assumes familiarity with general Spring Cloud Stream concepts, which can be found in its reference documentation.
 
-=== Overview
+== Introduction
 
-TBD
+Spring Cloud Stream Application Starters provide you with predefined Spring Cloud Stream applications that you can run independently or with Spring Cloud Data Flow.
+You can also use the starters as basis for creating your own applications.
+They include:
+
+* connectors (sources and sinks) for middleware including message brokers, storage (relational, non-relational, filesystem);
+* adapters for various network protocols;
+* generic processors that can be customized via SpEL or scripting.
```

Comment:
Link to [SpEL docs](https://docs.spring.io/spring/docs/current/spring-framework-reference/html/expressions.html)?